### PR TITLE
adding host as argument for gradio

### PR DESCRIPTION
**Pull request by jayshah96**

### CODE REVIEW

**Comment by tqchen** [comment]
cc @Kathryn-cat

File: python/mlc_chat/gradio.py [hunk 1]
```diff
@@ -30,6 +30,7 @@ def _parse_args():
                         is provided, it will be set to 0 by default.",
     )
     args.add_argument("--port", type=int, default=7860, help="The port number to run gradio.")
+    args.add_argument("--host", type=int, default="127.0.0.1", help="The port number to run gradio.")
```
Comment (Kathryn-cat) [inline python/mlc_chat/gradio.py:33]: Would you like to change "The port number to run gradio" into "The local host to run gradio" etc?

Comment (jayshah96) [reply python/mlc_chat/gradio.py]: @Kathryn-cat Done

**Comment by Kathryn-cat** [comment]
ready now

**Pull request merged**